CHANGELOG.md

```diff
@@ -29,12 +118,15 @@ All notable changes to Botonic will be documented in this file.
     - Added a new webchat setting [`storageKey`](https://github.com/hubtype/botonic/pull/1019) that indicates the key name to use in order to store the webchat state in `localStorage` (or `sessionStorage`).
 
 - [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
   - [Improvements](https://github.com/hubtype/botonic/pull/985) in Content.validate of text & shortText fields
+
     - Validate text.text also if content has no keywords (it could be accessed through a button)
     - Before, when shortText was empty, the contentful driver set it with the value of the name field (to ensure buttons did not show blank texts). Now this is managed by the Button model, so that MessageContent.validate() can report it.
```

Comment:
Before, when shortText was empty, the contentful driver set it with the value... -> Before, when shortText was empty, the Contentful driver used to set it with the value...